### ISSUE TITLE
SelectInput can't have a `label`/ `value` options

### ISSUE THREAD

**Issue by senecolas** [BRACKETS-by-TRIAD/craftable-pro-docs#22]
## Current
When using `SelectInput` component, we can only pass an array of string or number in `options` prop which is used both in value and  option label.

```vue
<SelectInput
    :options="['first_layout', 'second_layout']"
    v-model="form.layout_block"
    name="layout_block"
    label="Layout"
/>
```
<img width="415" alt="image" src="https://user-images.githubusercontent.com/35195832/230579999-2b424d31-e06e-4978-92b1-51ae98086a47.png">

If we want to differentiate `value` and `label`, we need to use `Multiselect` component which seems to overkill for a single selection.

```vue
<Multiselect
  :options="[
      {
          label: 'First Layout',
          value: 'first_layout',
      },
      {
          label: 'Second Layout',
          value: 'second_layout',
      },
  ]"
  :mode="'single'"
  v-model="form.layout_block"
  name="layout_block"
  label="Layout"
/>
```

## Proposal
Implement the same system of `options` of `Multiselect` inside `SelectInput`.

**Comment by timoransky** [comment]
Hi @senecolas!
Thank you for this proposal, again, it's very valid and something we completely missed.
Since this is not a big change, it might also be included in the next release coming in a few days.

**Comment by timoransky** [comment]
Hi @senecolas 
This feature was implemented in the new version ([1.1.7](https://docs.craftable.pro/changelog#v117---2023-04-13)). You can now use the `SelectInput` like this:
```
<SelectInput 
    :options="[
        {
            value: 'first_layout',
            label: 'First layout',
        },
        {
            value: 'second_layout',
            label: 'Second layout',
        },
    ]"
    v-model="form.layout_block"
    name="layout_block"
    label="Layout"
/>
```
And you can also specify the object keys like this (same as in `Multiselect` component):
```
<SelectInput 
    :options="[
        {
            foo: 'first_layout',
            bar: 'First layout',
        },
        {
            foo: 'second_layout',
            bar: 'Second layout',
        },
    ]"
    optionsLabel="bar"
    optionsValueProp="foo"
    v-model="form.layout_block"
    name="layout_block"
    label="Layout"
/>
```
